packages/components/MenuButton/README.md

```diff
@@ -0,0 +1,5 @@
+# MenuButton
+
+MenuButton is a wrapper control that combines and simplifies the API of Button and ContextualMenu.
+
+On macOS, it is implemented as a native module, and thus lacks the same level of customization as it's win32 counterpart.
```

Comment:
The way this is phrased makes it sound sort of negative, but I don't think it's all bad. We should generally provide the level of customization that the OS allows and if that's different from Windows, it doesn't mean the control is lacking support, it's just that it has the expected functionality for the platform

Reply:
We're limiting OS level customization too though. For example, I could have allowed you to set whatever `bezel` you want on the NSPopupButton to get different styles of menu buttons. I also don't allow you to use custom view as NSMenuItems (which is supported by Apple) with this approach. Ultimately, I chose to go with limited customization and a native module because the other approach (standing up FURN `ContextualMenu` on macOS) was experiencing difficulties, and we had already made this "helper" component for our client team that I could take advantage of and provide a native experience. 
Once this specific feature request is done, I still need to go back and stand up either `Callout` or `ContextualMenu` for future work, which means this implementation might be revisited.

Reply:
nit: its (no apostrophe)

Reply:
fixed